packages/react-dom-bindings/src/server/ReactDOMServerFormatConfig.js

```diff
@@ -123,6 +123,9 @@ const DataStreamingFormat: StreamingFormat = 1;
 export type ResponseState = {
   bootstrapChunks: Array<Chunk | PrecomputedChunk>,
   fallbackBootstrapChunks: void | Array<Chunk | PrecomputedChunk>,
+  requiresEmbedding: boolean,
+  hasHead: boolean,
+  hasHtml: boolean,
```

Comment:
I've force pushed a number of updates since this was in the commits. I think I sent you a commit range but that has been outdated for a while. I worry you probably reviewed a quite outdated version of the implemenation. For this bit of feedback in particular I already updated to a bitmask. I could consolidate further since I'm not using very many of the bits